tools/ci_build/github/linux/docker/scripts/install_ubuntu.sh

```diff
@@ -68,27 +106,54 @@ if [ $SYS_LONG_BIT = "64" ]; then
   rm -rf /tmp/dotnet
 fi
 
-if [ $PYTHON_VER != "3.5" ]; then
-    apt-get install -y --no-install-recommends \
-            python${PYTHON_VER} \
-            python${PYTHON_VER}-dev
-    update-alternatives --install /usr/bin/python3 python3 /usr/bin/python${PYTHON_VER} 1
-    update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.5 2
-    update-alternatives --set python3 /usr/bin/python${PYTHON_VER}
-    #TODO: the old one(/usr/bin/pip3) should be uninstalled first. Because the one will be
-    #put at /usr/local/. Then there will be two pips.
-    /usr/bin/python${PYTHON_VER} -m pip install --upgrade --force-reinstall pip==19.0.3
-fi
+if [ $OS_VERSION = "16.04" ]; then
+    if [ $PYTHON_VER != "3.5" ]; then
+        apt-get install -y --no-install-recommends \
+                python${PYTHON_VER} \
+                python${PYTHON_VER}-dev
+        update-alternatives --install /usr/bin/python3 python3 /usr/bin/python${PYTHON_VER} 1
+        update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.5 2
+        update-alternatives --set python3 /usr/bin/python${PYTHON_VER}
+        #TODO: the old one(/usr/bin/pip3) should be uninstalled first. Because the one will be
+        #put at /usr/local/. Then there will be two pips.
+        /usr/bin/python${PYTHON_VER} -m pip install --upgrade --force-reinstall pip==19.0.3
+    fi
 
-/usr/bin/python${PYTHON_VER} -m pip install --upgrade --force-reinstall numpy==1.15.0
-/usr/bin/python${PYTHON_VER} -m pip install --upgrade --force-reinstall requests==2.21.0
-if [ $DEVICE_TYPE = "Normal" ]; then
-    /usr/bin/python${PYTHON_VER} -m pip install --upgrade --force-reinstall sympy==1.1.1
-fi
-/usr/bin/python${PYTHON_VER} -m pip install --upgrade scipy
-rm -rf /var/lib/apt/lists/*
+    /usr/bin/python${PYTHON_VER} -m pip install --upgrade --force-reinstall numpy==1.15.0
+    /usr/bin/python${PYTHON_VER} -m pip install --upgrade --force-reinstall requests==2.21.0
+    if [ $DEVICE_TYPE = "Normal" ]; then
+        /usr/bin/python${PYTHON_VER} -m pip install --upgrade --force-reinstall sympy==1.1.1
+    fi
+    /usr/bin/python${PYTHON_VER} -m pip install --upgrade scipy
+    rm -rf /var/lib/apt/lists/*
 
-if [ $DEVICE_TYPE = "Normal" ]; then
-aria2c -q -d /tmp -o llvm.tar.xz http://releases.llvm.org/9.0.0/clang+llvm-9.0.0-x86_64-linux-gnu-ubuntu-16.04.tar.xz
-tar --strip 1 -Jxf /tmp/llvm.tar.xz -C /usr
+    if [ $DEVICE_TYPE = "Normal" ]; then
+        aria2c -q -d /tmp -o llvm.tar.xz http://releases.llvm.org/9.0.0/clang+llvm-9.0.0-x86_64-linux-gnu-ubuntu-16.04.tar.xz
+        tar --strip 1 -Jxf /tmp/llvm.tar.xz -C /usr
+    fi
+else
+    if [ $PYTHON_VER != "3.6" ]; then
```

Comment:
!= 3.7 i thought. since 3.7 is the default python version for ubuntu 18.04

Reply:
but this seems to be working? Need to look at the build logs.

Reply:
default python version in ubuntu18.04 is v3.6

Reply:
yes, you're right. out of box it's 3.6. looks like 3.7.1 is in the updates repo.